deployment/v2/4_createRollup.ts

```diff
@@ -64,19 +64,28 @@ async function main() {
         adminZkEVM,
         forkID,
         consensusContract,
+        dataAvailabilityProtocol,
     } = createRollupParameters;
 
     const supportedConensus = [
         "PolygonZkEVMEtrog",
         "PolygonZkEVMV2",
-        "PolygonDataComittee",
-        "PolygonDataComitteeEtrog",
+        "PolygonValidium",
+        "PolygonValidiumEtrog",
     ];
 
     if (!supportedConensus.includes(consensusContract)) {
         throw new Error(`Consensus contract not supported, supported contracts are: ${supportedConensus}`);
     }
 
+    const supporteDataAvailabilityProtocols = [
+        "PolygonDataCommittee",
```

Comment:
Shouldn't it be `PolygonValidium`? Because `PolygonDataCommittee` is renamed to `PolygonValidium` earlier.

There are other instances where this is being used and also the word `comittee`

Reply:
hey!
thanks for reviewing the PRs! ^^
It's ok, take in account that this makes refrence to the `supporteDataAvailabilityProtocols` , so which contracts will verify the callData of the Validium
`PolygonValidium` will be the contract where the Batches are sequenced, while `PolygonDataCommittee` the contract that will validate that data  against a data commitee^^